bsip-0075.md

```diff
@@ -20,16 +20,16 @@ Asset owners are often in the best position to control the values of MCR and MSS
 # Rationale
 Prior to this change, adjustments to MCR and MSSR were handled by feed producers. This often requires the asset owner to contact the feed producer and ask for the change. For some assets, such a step is time consuming for both parties and ill-fitted as part of the feed producer's responsibilities.
 
-This new change shall permit the asset owner to determine at asset creation time which should have the responsibility to maintain those values.
+This new change shall permit the asset owner to determine *at asset creation time* who should have the responsibility to maintain those values.
 
 # Specification
-The `asset_object.asset_options.flags` shall contain the new flag `issuer_fed_mcr` that determines if the MCR value is stored as part of the asset_object, or if it is pulled from the feeds.
+The `asset_object.asset_options.issuer_permissions` shall contain the new `issuer_fed_mcr` permission that determines if an issuer can change the related `flags` value.
 
-The `asset_object.asset_options.flags` shall contain the new flag `issuer_fed_mssr` that determines if the MSSR value is stored as part of the asset_object, or if it is pulled from the feeds.
+The `asset_object.asset_options.issuer_permissions` shall contain the new `issuer_fed_mssr` permission that determines if an issuer can change the related `flags` value.
 
-The `create_asset_operation` shall be modified to include the setting of the parameters above, as well as initial values for MCR and MSSR.
+The `create_asset_operation` shall be modified to include the setting of the parameters above, as well as initial `flags` values for MCR and MSSR.
 
-The `update_asset_operation` shall be modified permit updating of the MCR and MSSR values, if the asset has been created with the permission for the asset owner to do so. NOTE: much of the logic currently in `asset_publish_feeds_evaluator::do_apply()` (i.e. call order checks and calculations) shall be also called here if MCR or MSSR are adjusted.
+The `update_asset_operation` shall be modified to permit updating of the MCR and MSSR values, if the asset has the appropriate `flags` value set permitting the asset owner to do so. NOTE: much of the logic currently in `asset_publish_feeds_evaluator::do_apply()` (i.e. call order checks and calculations) shall be also called here if MCR or MSSR are adjusted.
```

Comment:
I think that the prior text, which references the pre-existing permission setting of the asset, is more correct